src/styles/tailwind/list.ts

```diff
@@ -0,0 +1,7 @@
+export type Variant = 'pink' | 'grey' | 'base';
+
+export const listVariants: Record<Variant, string> = {
+	pink: 'bg-[#FFEBE7]',
+	grey: 'bg-[#F7F7F7]',
+	base: 'bg-white'
+};
```

Comment:
```ts
import { tv, type VariantProps } from 'tailwind-variants';

export const listVariants = tv({
	base: 'rounded-lg p-4',
	variants: {
		variant: {
			pink: 'bg-sucu-pink-500',  
			grey: 'bg-sucu-gray-light',  
			default: 'bg-white'
		}
	},
	defaultVariants: {
		variant: 'default'
	}
});

export type Variant = VariantProps<typeof listVariants>['variant'];
```

i prefer this way using `tailwind-variant` lib for many variants component bc. it's easier to scale and to make complex components, but it's up to you this is just my pref. 😆. @punchanabu what do you think?

Reply:
Oh, could you please change the color to match what’s in Figma?

Reply:
Sory, I mean the preset color in tailwind config

Reply:
okok

Reply:
go use the `tailwind-variant` เลย kub